deepcave/config.py

```diff
@@ -117,17 +119,17 @@ def PLUGINS(self) -> Dict[str, List[Any]]:
             ],
             "Objective Analysis": [
                 CostOverTime(),
-                ConfigurationCube(),
                 ParetoFront(),
-                ParallelCoordinates(),
             ],
             "Budget Analysis": [
                 BudgetCorrelation(),
             ],
             "Hyperparameter Analysis": [
                 Importances(),
                 AblationPaths(),
+                ConfigurationCube(),
                 PartialDependencies(),
+                ParallelCoordinates(),
```

Comment:
Just a really minor thing, but I'd move PartialDependencies to Symbolic Explanations, as they are somewhat similiar.